src/doc/reference.md

```diff
@@ -1413,6 +1413,21 @@ a = Animal::Cat { name: "Spotty".to_string(), weight: 2.7 };
 In this example, `Cat` is a _struct-like enum variant_,
 whereas `Dog` is simply called an enum variant.
 
+Enums can have a discriminator:
```

Comment:
Enums always have a discriminant, this syntax just lets you specify exactly what it is. They'll default to 0, 1, 2, ...